nextflow.config

```diff
@@ -22,9 +22,15 @@ params.region_file = ""
 // calling stilts
 params.stilts = "stilts"
 
+// calling swarp
+params.swarp = "SWarp"
+
 // Source finding params
 params.region_file = "square.mim"
 
+// Plotting options
+params.plotdates = true
```

Comment:
This we think is the origin of:

`WARN: Access to undefined parameter `dates` -- Initialise it to a default value eg. `params.dates = some_value`

Reply:
Fixed in f9bc27b